bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -22,9 +24,7 @@ Gem::Specification.new do |spec|
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files         = Dir.chdir(File.expand_path("..", __FILE__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  end
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
```

Comment:
Please add back the `Dir.chdir` block. Without that, evaluating the gemspec from outside `File.expand_path("..", __FILE__)` won't work.

Reply:
Yikes!
Then I shall just remove the spaces :confused:

Reply:
You can do this

```ruby
spec.files         = Dir.chdir(File.expand_path("..", __FILE__)) { `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) } }
```

Reply:
The line length exceeds the usual limit. So since there's no way can we wrap this within 125 characters, I'll remove the spaces for now \o/